### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by bhaskardabhi-oviyum**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/bhaskardabhi-oviyum/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

**Comment by bhaskardabhi-oviyum** [comment]
hi

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (bhaskardabhi-oviyum) [inline ReadMe.md:1]: Remove it

Comment (bhaskardabhi-oviyum) [inline ReadMe.md:1]: !resolve

Comment (bhaskardabhi-oviyum) [inline ReadMe.md:1]: !fixed

**Pull request merged**